src/parsec_client/operations/psa_export_key.md

```diff
@@ -0,0 +1,57 @@
+# PsaExportKey
+
+Export a key in binary format. Opcode: 12 (`0x000C`)
+
+## Parameters
+
+| Name       | Type   | Description               |
+|------------|--------|---------------------------|
+| `key_name` | String | Name of the key to export |
+
+## Results
+
+| Name   | Type            | Description                    |
+|--------|-----------------|--------------------------------|
+| `data` | Vector of bytes | Buffer containing the key data |
+
+## Specific response status codes
+
+- `PsaErrorInvalidArgument`: The key is not a valid key.
+
+## Description
+
+The output of this function can be passed to [PsaImportKey](psa_import_key.md) to create an object
+that is equivalent to key.
+
+For standard key types, the output format is as follows:
+
+- For symmetric keys, including [`MAC`](psa_key_attributes.md#hmac-type) keys, the format is the raw
+   bytes of the key.
+- For [`DES`](psa_key_attributes.md#des-type), the key data consists of 8 bytes. The parity bits
+   must be correct.
+- For [`Triple-DES`](psa_key_attributes.md#des-type), the format is the concatenation of the two or
+   three DES keys.
+- For RSA key pairs, with key type [`RsaKeyPair`](psa_key_attributes.md#rsakeypair-type), the format
+   is the non-encrypted DER encoding of the representation defined by PKCS#1 in RFC 8017 as
```

Comment:
Could you add a link here for [RFC 8017](https://tools.ietf.org/html/rfc8017#appendix-A.1.2) and one below for [RFC 5915](https://tools.ietf.org/html/rfc5915), like the one for RFC 7748?